test/PaymentTest.py

```diff
@@ -12,6 +13,203 @@ class TestPayments(unittest.TestCase):
     client.password = "YourWSPassword"
     client.platform = "test"
     client.app_name = "appname"
+    client.xapikey = ""
+
+    def test_authorise_success_mocked(self):
```

Comment:
Hi @Ishrock ,
thank you for your PR!
Are these duplicate tests?

Reply:
Yes and no.  I run the same tests that are run on the client for basic auth, just with the xapi key and no basic auth instead to ensure they would pass with a diff authentication.  I tried to find a dryer way to do it with subclassing the test case but it did not work out as expected.